a3p-integration/proposals/z:acceptance/invitation-test-submission/send-script.tjs

```diff
@@ -0,0 +1,27 @@
+/* global E */
+
+/// <reference types="@agoric/vats/src/core/core-eval-env"/>
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+
+/**
+ * Send a payment by looking up deposit facet via namesByAddress.
+ *
+ * see ./post.test.js
+ *
+ * @param {BootstrapPowers} powers
+ */
+const sendIt = async powers => {
```

Comment:
hm. what's misleading about the comment? The purpose of the function is to verify that namesByAddress is working.

The fact that the payment is an invitation to the reserve is incidental. It's just an easy payment to construct.

Reply:
> The purpose of the function is to verify that namesByAddress is working.

> The fact that the payment is an invitation to the reserve is incidental. It's just an easy payment to construct.

In that case a few comments would clarify. 
 * mention that this file is used as a template, and that **ADDRESS** is edited before use
 * mention that the use of the reserve is incidental since the purpose is testing `namesByAddress`.

Reply:
Changes addressed on commit [e0fa6c3](https://github.com/Agoric/agoric-sdk/pull/10123/commits/e0fa6c35b3898f6bc5dd30659754545953cea371)